test/Driver/linker.swift

```diff
@@ -19,6 +19,9 @@
 // RUN: %swiftc_driver -sdk "" -driver-print-jobs -target x86_64-unknown-linux-gnu -Ffoo -Fsystem car -F cdr -framework bar -Lbaz -lboo -Xlinker -undefined %s 2>&1 > %t.linux.txt
 // RUN: %FileCheck -check-prefix LINUX-x86_64 %s < %t.linux.txt
 
+// RUN: %swiftc_driver -sdk "" -driver-print-jobs -target x86_64-unknown-linux-gnu -Ffoo -Fsystem car -F cdr -framework bar -Lbaz -l boo -Xlinker -undefined %s 2>&1 > %t.linux.txt
```

Comment:
Given that this test is turned off right now, instead of duplicating the test, maybe you can add a FIXME next to the `REQUIRES` at the top, say something like:

```
// FIXME: When this is turned on, please move the test from linker-library-with-space.swift
// to this file and remove that file.
```